### ISSUE TITLE
What is the status of this repo?

### ISSUE THREAD

**Issue by xuhdev** [editorconfig/editorconfig-defaults#14]
I'm very confused. The repo doesn't state how it should be used.

**Comment by xuhdev** [comment]
ping @ffes Do you think we need to add something to README?

**Comment by ffes** [comment]
You're right. At the moment it is not clear. I personally think it could become a useful source for plugin authors to add snippets to a `.editorconfig` file, among others. I'm sure others have much better ideas to utilize the data.

But at the moment there are not much defaults in the list and the file format is not finalized as well.

And thanks for the ping. I more or less forgot about this repo.

**Comment by xuhdev** [comment]
Actually someone asked something related to this repo in the mailing list: https://groups.google.com/d/msg/editorconfig/0QOiB3DjUl4/pl2AJAOdBQAJ

**Comment by ffes** [comment]
> Do you think we need to add something to README?

Yes! I bit clarification what the intent of the repo is, is surely a good thing to do. I will pickup #10 again to finalize the format. Maybe wait for that because the PR does include changes to the README.

Once that is done, it is just a matter of adding more snippets. And get the attention to plugin developers to include a snippet feature. For Notepad++ I will definitely look into that myself.

I also have some code lying around to include the content of `editorconfig-defaults.json` in the [main website](https://editorconfig.org). This way the snippets will become more visible as well.

But as said, first resolve the conflict and finalize the format as I tried with #10
